library/ssl_tls.c

```diff
@@ -109,6 +109,33 @@ static void ssl_update_in_pointers( mbedtls_ssl_context *ssl,
 
 #if defined(MBEDTLS_SSL_PROTO_DTLS)
 
+/* Top-level Connection ID API */
+
+int mbedtls_ssl_set_cid( mbedtls_ssl_context *ssl,
+                         int enable,
+                         unsigned char const *own_cid,
+                         size_t own_cid_len )
+{
+    ((void) ssl);
+    ((void) enable);
+    ((void) own_cid);
+    ((void) own_cid_len);
+    return( 0 );
+}
+
+int mbedtls_ssl_get_peer_cid( mbedtls_ssl_context *ssl,
```

Comment:
Please add a big fat warning to the documentation of the function that so far it does nothing, except always set `*enabled` to `DISABLED`.

Reply:
Yes - as above.

Reply:
Done.